### ISSUE TITLE
New endpoint: owned safes across networks

### ISSUE THREAD

**Issue by katspaugh** [safe-global/safe-client-gateway#449]
## Overview
For the "In-app Network Switching" epic, we'll need to display the safes owned by a user across multiple networks in the app sidebar.

## Requirements
We'd like to be able to receive is a **list** of records like this:
```
[{
  "address": "0x79E0ec6c7AecFecC7D03E8565837fB5Bf0BB691f",
  "chainId": "4"
  "dateCreated": "2021-06-21"
},
{
  "address": "0x79E0ec6c7AecFecC7D03E8565837fB5Bf0BB691f",
  "chainId": "1"
  "dateCreated": "2021-06-29"
}]
```

The request parameters should be:
 * `ownerAddress`
 * `chainIds` – comma-separated list of chain ids

☝️ `chainIds` is supposed to reduce the scope of the request to not fetch safes on _all_ the networks.

## Existing endpoints
There's already a transaction-service endpoint to fetch the list of owner's safes but it returns only the addresses:

https://safe-transaction.rinkeby.gnosis.io/api/v1/owners/0x474e5Ded6b5D078163BFB8F6dBa355C3aA5478C8/safes/

## Screens
<img width="964" alt="Screenshot 2021-06-29 at 15 21 37" src="https://user-images.githubusercontent.com/381895/123804690-b7f3f780-d8ed-11eb-852b-42422a2b601a.png">

[Figma screens](https://www.figma.com/proto/txqgq1QhrFFe2NlwPRh0bz/Gnosis-Safe-Web---2021-02-05?node-id=519%3A97&viewport=799%2C302%2C0.35462266206741333&scaling=min-zoom&page-id=519%3A0%3Ffuid%3D789860674698606135&hide-ui=1)

**Comment by jpalvarezl** [comment]
Judging by the discussion on slack, it seems that this issue should be on `not now` but maybe there is still something that needs to be mentioned on the subject, therefore left it in `specs to review`

**Comment by katspaugh** [comment]
Update: added a `chainId` parameter in the specs, to make the endpoint "lighter".
We'll pass a list of chains that the user has used.

**Comment by sche** [comment]
As part of the [Owner keys are confusing and not safe to use](https://github.com/gnosis/safe-pm/issues/55) problem statement, we want to implement suggesting to import owned safes when importing a key by a seed phrase.
We would like to start working on [improving key import flow](https://github.com/gnosis/safe-ios/issues/2208) in roughly three weeks. Would it be possible to implement this endpoint in the following 2-4 weeks?

**Comment by rmeissner** [comment]
I would not do this before we have https://github.com/gnosis/safe-transaction-service/issues/763 in place. Once it is done the performance overhead should be manageable.

**Comment by katspaugh** [comment]
Please feel free to close this, as we currently only display owned safes on a given chain.

**Comment by fmrsabino** [comment]
Will do. We can reopen later it needed.

Closing as not planned.
